streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapter.java

```diff
@@ -1396,5 +1396,6 @@ public void close() {
         // ColumnFamilyOptions should be closed last
         dbOptions.close();
         columnFamilyOptions.close();
+        super.close();
```

Comment:
Are there any ordering constraints to call the three `close` methods?

Reply:
I don't think it matters in this case as the parent's Options handle isn't actually used. Still, I've maintained the inverse of the initialization order here.

Reply:
updated the comment